eng/pipelines/common/global-build-job.yml

```diff
@@ -54,6 +54,11 @@ jobs:
         ${{ if ne(parameters.isOfficialBuild, true) }}:
           value: ''
 
+      - ${{ if and(eq(parameters.osGroup, 'Android'), eq(parameters.isOfficialBuild, false)) }}:
+        - name: ANDROID_OPENSSL_AAR
+          value: /tmp/openssl-android
+
+      - ${{ parameters.variables }}
```

Comment:
This is wrong, we're doing the same with the foreach below, basically propagating the parameters.variables that are an array.